webpack/globalize.ts

```diff
@@ -0,0 +1,5 @@
+import * as clarity from "../src/clarity";
```

Comment:
Why is this file needed for webpack? Can you add a comment to explain that?

Reply:
Webpack bundle doesn't store the target module in a global variable by default like rollup used to do for us, so we need an extra wrapper module that does this manually.

---
In reply to: [250238167](https://github.com/Microsoft/clarity-js/pull/126#discussion_r250238167) [](ancestors = 250238167)

Reply:
Added comment as well

---
In reply to: [250333677](https://github.com/Microsoft/clarity-js/pull/126#discussion_r250333677) [](ancestors = 250333677,250238167)